backend/curriculum_tracking/tests/test_UserStatsPerWeekSerializer.py

```diff
@@ -0,0 +1,219 @@
+            UserStatsPerWeekSerializer, User=assigned_person
+        ) == 1
+
+    def test_cards_in_the_review_column_waiting_for_a_review(self):
+
+        # Two cards in the IN_REVIEW column
+        content_item_3 = ContentItemFactory(
+            content_type=ContentItem.TOPIC,
+            project_submission_type=ContentItem.REPOSITORY,
+        )
+
+        card_3 = AgileCardFactory(
+            content_item=content_item_3,
+            status=AgileCard.IN_REVIEW,
+            recruit_project=None,
+        )
+
+        card_4 = AgileCardFactory(
+            content_item=content_item_3,
+            status=AgileCard.IN_REVIEW,
+            recruit_project=None,
+        )
+
+        assigned_person = SocialProfileFactory().user
+        card_3.assignees.set([assigned_person])
+        card_4.assignees.set([assigned_person])
+
+        # We should get two cards in the review column waiting for a review
+        assert UserStatsPerWeekSerializer.get_cards_in_review_column(
+            UserStatsPerWeekSerializer, User=assigned_person
+        ) == 2
+
+    def test_pr_reviews_card_in_different_column_due_to_review_status(self):
+
+        # Now creating reviews on the PR
+        pr_review_for_pr_today = PullRequestReview.objects.create(
+            html_url="https://github.com/Umuzi-org/PullRequest-Review(1)",
+            pull_request_id=self.pull_request.id,
+            author_github_name="DeadManWalking",
+            submitted_at=self.today,
+            body="A horse walks into a bar, the barman says 'Why the long face?'",
+            commit_id=self.repo_card_one.assignees.first().id,
+            state="closed",
+            user_id=self.repo_card_one.assignees.first().id
+        )
+        pr_review_for_pr_today.save()
+
+        pr_review_for_pr_yesterday = PullRequestReview.objects.create(
+            html_url="https://github.com/Umuzi-org/PullRequest-Review(2)",
+            pull_request_id=self.pull_request.id,
+            author_github_name="DeadManWalking",
+            submitted_at=self.yesterday,
+            body="A horse walks into a bar, the barman says 'Why the long face?'",
+            commit_id=self.repo_card_one.assignees.first().id,
+            state="closed",
+            user_id=self.repo_card_one.assignees.first().id
+        )
+        pr_review_for_pr_yesterday.save()
+
+        pr_review_two_weeks_ago = PullRequestReview.objects.create(
+            html_url="https://github.com/Umuzi-org/PullRequest-Review(3)",
+            pull_request_id=self.pull_request.id,
+            author_github_name="DeadManWalking",
+            submitted_at=self.two_weeks_ago,
+            body="This review happened a long time ago.",
+            commit_id=self.repo_card_one.assignees.first().id,
+            state="closed",
+            user_id=self.repo_card_one.assignees.first().id
+        )
+        pr_review_two_weeks_ago.save()
+
+        pr_review_two_days_before_yesterday = PullRequestReview.objects.create(
```

Comment:
All these object creations seem quite verbose. It would be better to use factoryboy to create a factory and then use the factory. Then other people can also use the factory in their tests. It saves time in the medium term